uktrade/data-hub-frontend
# Cls2 561 tasks assigned to me reminders

## Description of change

Added `task assigned to me from others` to reminders UI in notification bell

## Test instructions

With the sandbox environment running:
Navigate to `/reminders`
You should see `My Tasks` in the left hand panel with `Task Assigned To Me From Others`
Select this to see example sandbox task reminders

With the dev environment running:
Navigate to `/reminders`
You should see `My Tasks` in the left hand panel with `Task Assigned To Me From Others`
Add a `task assigned to me from others reminder` in the django backend and it should appear here

## Screenshots
![Screenshot 2023-11-08 at 16 37 52](https://github.com/uktrade/data-hub-frontend/assets/54268863/5b077d5c-b66a-4556-8238-6562f3a9b349)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
